### DISCUSSION TITLE
How to ctrl/shift + click in version 14+

### DISCUSSION THREAD

**Discussion by rvsia** [testing-library/user-event#942] [Q&A] [answered]
Hello,

previously, it was possible to ctrl+click or shift+click using `ctrlKey: true` or `shiftKey: true` in the `userEvent.click` options ([read here](https://testing-library.com/docs/ecosystem-user-event/#clickelement-eventinit-options))

However, these options have been removed in the version 14.

So I am trying to do:

```jsx
    await userEvent.keyboard('{Control>}');
    await userEvent.click(screen.getByText('zebras'));
    await userEvent.keyboard('{/Control}');
```

but it doesn't work. `ctrlKey` is set to `false` in the event.

How I can do it properly? 🤔 

Thanks,
Richard

**Comment by rvsia** [comment]
Also tried:

```jsx
    await userEvent.pointer([
      { target: screen.getByText('zebras') },
      { keys: '{Control>}[MouseLeft]{/Control}', target: screen.getByText('zebras') },
    ]);
```

**Reply by ph-fritsche** [reply]
The `pointer` API only accepts [pointer device keys](https://testing-library.com/docs/user-event/pointer#pressing-a-button-or-touching-the-screen).

**Comment by ph-fritsche** [comment, marked as answer]
You need to [start a session](https://testing-library.com/docs/user-event/setup#starting-a-session-per-setup). (This is also [the recommended way to use this library](https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent) in `v14`.)

If you call the methods on the default export, the calls each act on their own input device state.

**Reply by rvsia** [reply]
Thanks!

Yeeah, I am not using the sessions yet because I manage hundreds of tests and even just refactoring them to compatible with version 14 takes a lot of time. 😅
